cmd/release/cmd/generate.go

```diff
@@ -99,7 +99,7 @@ var k3sGenerateTagsSubCmd = &cobra.Command{
 		version := args[0]
 		k3sRelease, found := rootConfig.K3s.Versions[version]
 		if !found {
-			return NewVersionNotFoundError(version)
+			return NewVersionNotFoundError(version, "k3s")
```

Comment:
should we have constants for these values "k3s", "rancher", etc? iirc there's some defined at `cmd/release/config/config.go`